doc/contributing/releases.md

```diff
@@ -545,6 +546,20 @@ with the Build Working Group.
 
 </details>
 
+#### 7.1 Updating the release _(optional)_
+
+Sometimes a release might be deferred to the subsequent day due to several conditions:
+
+* Unstable CI
+* Bad commit, requiring a git bisect and updating the proposal
+* Late CI completion
+
+And when it happens, the CHANGELOG_Vx and the commit metadata needs to be updated
+according to the new target date.
+
+However, if it's just the changelog/commit message that has changed since the last CI
+execution, there's no need to rerun CI, V8, or CITGM workflows.
```

Comment:
Perhaps clarify that the updated PR still needs a clean GitHub actions run (i.e. to validate the updated changelog)?

Reply:
I think also clarify that the original CI, V8 and CITGM run urls should be in a visible comment (and not hidden).

Reply:
Good one. I'll update.